README.md

```diff
@@ -1,39 +1,82 @@
+3. The deposit gets automatically swapped to the interest-accruing equivalent on Aave, and is stored in the `Vault`.
+4. The depositor gets minted the project's `bToken`, redeemable for underlying funds when they `withdraw` their principal (and any additional interest) from the `Vault`
+5. Interest on deposits accrues to the project's `Vault`. Every 30 days, the accrued interest gets allocated to be `recipient`, available to be withdrawn.
+6. If the interest accrued from the sum of deposited funds exceeds the recipient's desired monthly `salary` for that month, additional interest earned for the remainder of that month gets allocated to depositors.
+7. Recipients can `withdraw` a specified `amount` at any time.
+8. Depositors can `withdraw` their principal (and any additional interest) at any time.
+
+Note: The `Vault` has sole access to `mint` and `burn` on its associated `bToken`, as these functions need only be called during `deposit` and `withdraw`
+
+### bToken Exchange Rate
+
+The 1-1 exchange rate only applies to the initial deposit because any subsequent deposits has to take into account (1) the interest already accrued and (2) whether salary goals have been met. Each bToken is convertible into a dynamic quantity of the underlying asset, as depositors deposit and withdraw from the project's `Vault`.
+
+The exchange rate between a bToken and the underlying asset is calculated as follows:
+
+`exchangeRate = principal + surplusInterest / bTokenTotalSupply`
+
+## 👩🏻‍💻 Development
+
+### Prerequisites
+- Node v10.21.0
+- Solidity v0.6.12 (solc-js)
+
+### Setup
+- Clone the repo using `git clone https://github.com/bazaarfinance/bazaarfinance.git && cd bazaarfinance`
+- Install deps with yarn `yarn` or npm `npm install`
+- To start the client locally, start hardhat `npx hardhat node --watch`
+- From a new terminal window and install dependencies `cd frontend && yarn install` or `cd frontend && npm install`
+- Import seed phrase in Metamask. The default mnemonic currently used by hardhat is `test test test test test test test test test test test junk`
+- Ensure Metamask RPC is set to `http://localhost:8545` and chainID `31337`.
+- Run `yarn start` or npm `npm start` and a new browser window should open up at http://localhost:3000/
 
-Ensure you are useing RPC to http://localhost:8545.
+## ✅ Testing
+- Run unit tests `npx hardhat test`
+- Run integration tests `npx hardhat --config hardhat.integration-test-config.ts test ./test/integration/*.ts` from the main directory
 
-You may also need to set the chainID to 31337 if you are useing Hardhat blockchain development node.
+Since the tests fork mainnet for interaction with AAVE, you'll need to interact with a node service such as [Alchemy](http://alchemyapi.io/), which requires an API Key, for tests to work. See `.env-sample` for instructions on setting up a local `.env` file that stores the mainnet `FORKING_URL`
```

Comment:
moved all example env files to env-example-files in https://github.com/bazaarfinance/bazaarfinance/pull/14.

`.env-sample` --> `env-example-files/.env-example`